apps/src/pythonlab/PythonlabView.tsx

```diff
@@ -166,6 +167,11 @@ const PythonlabView: React.FunctionComponent<
     restartPyodideIfProgramIsRunning
   );
 
+  // Set view code to false if level is switched for any levels in widget view.
+  useLifecycleNotifier(LifecycleEvent.LevelLoadStarted, () => {
```

Comment:
When a user switches from a widget-view level with show/hide code to another widget-view level with show/hide code, this ensures the code editor is hidden at level load.